cdap-api/src/main/java/co/cask/cdap/api/annotation/Capabilities.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.cdap.api.annotation;
+
+/**
+ * Defines the different capability options for {@link Plugin} to specify.
+ */
+public final class Capabilities {
+  public static final String NATIVE = "native"; // defines capability of running in native
```

Comment:
The comment should be a javadoc for the public constant.

Reply:
Done.